controllers/userController.js

```diff
@@ -12,7 +12,7 @@ const signUp = async (req, res) => {
       throw error;
     }
 
-    if (typeId !== 1 && !account) {
+    if (type_id !== 1 && !account) {
```

Comment:
해당 수정 사항이 한 두번이 아닌 것 같은데, push 하고 fileChanged 확인 안 하시나요?
```suggestion
    if (typeId !== 1 && !account) {
```